### ISSUE TITLE
Future Request: file sorting options

### ISSUE THREAD

**Issue by yurix4** [sumatrapdfreader/sumatrapdf#541]
Different options of file sorting (type, name, number, date, etc) in image folders and CBR/CBZ files will makes Summatra better as image viewer.

**Comment by GitHubRulesOK** [comment]
@kjk 
SumatraPDF only requires **sort by name in natural order**, and we had enough problems just ensuring that was working :-) I see little benefit in introducing further complications by maintaining choices and the necessary controls to effect them, its inevitably up to you (or another developer) to add them but I suggest this could be categorized as "won't fix further"

**Comment by SumatraPeter** [comment]
@GitHubRulesOK: I agree. Unless Sumatra implements a file manager type interface in future where such sorting options might come in handy, can't see much use for them when simply viewing images in comics (CRx) etc. where they're supposed to be viewed in a certain fixed order.

**Comment by kjk** [comment]
Closing as won't-fix.

SumatraPDF is a document/page reader, not a file manager. For image folders and CBR/CBZ comics, pages have a single intended reading order, so SumatraPDF always sorts them by **file name in natural order** (numeric chunks compare by value, so `page2.jpg` sorts before `page10.jpg`). That's the ordering comics and scanned-page folders need.

Other sort criteria (date, type, size) don't make sense for reading pages in sequence — they'd just scramble the order. Supporting selectable orders would add UI, a persisted setting, and re-sort logic for little benefit, which matches the discussion here. The one ordering that matters — sensible natural-name sort — already works.
